### ISSUE TITLE
Git Credentials wrong, Failed to push some refs

### ISSUE THREAD

**Issue by dmarti03x1** [desktop/desktop#2887]
### Description
I'm trying to push some code into my GitHub Repo. 
I can commit and I can Clone and open in GithubDesktop but I can't PUSH to origin.
So far it only happens on my laptop and have tried it on another laptop and everything functions normal.
I have tried to fresh install, sign in and sign out, and even reset my password.
I have NEVER messed around with any of the program files for GitHubDesktop.

### Version

**GitHub Desktop version:** [1.0.1]

**OS version:** [Windows 10.0.15063]

### Steps to Reproduce

1. [edit code and save from a text editor (Notepad++)]
2. [Commit with Summary and Description]
3. [Click, PUSH ORIGIN]

**Expected behavior:** [Expect to push to my GitHub Repo]

**Actual behavior:** [Error message: Exit status 1]

**Reproduces how often:** [All the time
[2017-09-26.desktop.production.log](https://github.com/desktop/desktop/files/1335364/2017-09-26.desktop.production.log)
]

#### Additional Information
I have started getting issues when I initially wanted to push a video to my GitHub repo. I got a 100mb error and tried to compress the video to upload. Eventually I gave up because the video was to big and removed video but then was getting the same error even though it was removed. I don't get that 100mb error anymore.

**Comment by shiftkey** [comment]
@dmarti03x1 this is a duplicate of #2812, and we're waiting on an update to Git LFS to close that out. Please follow along with that issue.
